deploy/helm-chart/harbor-operator/templates/instance/registry.yaml

```
@@ -0,0 +1,41 @@
+{{- $this := . }}
+{{- range $instance := .Values.instances }}
+  {{- range $registry := .registries }}
+apiVersion: registries.mittwald.de/v1alpha1
+kind: Registry
+metadata:
+  name: {{ $registry.name }}
```

Can there be two instances with registries of the same name? If yes, maybe...

```suggestion
  name: {{ $instance.name }}-{{ $registry.name }}
```

❓